nexpose/nexpose.py

```diff
@@ -620,6 +625,13 @@ def RequestReportDelete(self, report_id, reportconfiguration_id=0):
         else:
             return self.ExecuteBasicOnReport("ReportDeleteRequest", report_id)
 
+    def RequestAdhocReport(self, report_config):
```

Comment:
I don't quite understand why this is a new function...? Couldn't we just re-use the one below? Or do you want to make breaking changes here that wouldn't work with the below function?

Reply:
I can't really remember why I made new ones, but they don't need to be new. I am not terribly concerned about backwards compatibility either at this point since there is a lot of work needed to get this to be on par with the ruby gem for functionality.